src/components/NcEmojiPicker/NcEmojiPicker.vue

```diff
@@ -145,23 +145,37 @@ This component allows the user to pick an emoji.
 			:per-line="8"
 			:picker-styles="{ width: '320px' }"
 			:show-preview="showPreview"
+			:skin="currentSkinTone"
+			:show-skin-tones="false"
 			:title="previewFallbackName"
 			role="dialog"
 			:aria-label="t('Emoji picker')"
 			v-bind="$attrs"
 			@select="select">
 			<template #searchTemplate="slotProps">
-				<NcTextField ref="search"
-					class="search"
-					:value.sync="search"
-					:label="t('Search')"
-					:label-visible="true"
-					:placeholder="i18n.search"
-					trailing-button-icon="close"
-					:trailing-button-label="t('Clear search')"
-					:show-trailing-button="search !== ''"
-					@trailing-button-click="clearSearch(); slotProps.onSearch(search);"
-					@update:value="slotProps.onSearch(search)" />
+				<div class="search__wrapper">
+					<NcTextField ref="search"
+						class="search"
+						:value.sync="search"
+						:label="t('Search')"
+						:label-visible="true"
+						:placeholder="i18n.search"
+						trailing-button-icon="close"
+						:trailing-button-label="t('Clear search')"
+						:show-trailing-button="search !== ''"
+						@trailing-button-click="clearSearch(); slotProps.onSearch(search);"
+						@update:value="slotProps.onSearch(search)" />
+					<NcColorPicker palette-only
+						:palette="skinTonePalette"
```

Comment:
Missing a container here for inner NcPopover. Because of that, skin tones are rendered always on 'body'